src/System.Net.Requests/tests/AuthenticationManagerTest.cs

```diff
@@ -12,29 +12,41 @@ namespace System.Net.Tests
 {
     public class AuthenticationManagerTest
     {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "AuthenticationManager supported on NETFX")]
```

Comment:
Is there a test that verifies this works on .Net Framework? Are we now porting all tests from .Net Framework here?